csrc/kernels_hip.cuh

```diff
@@ -0,0 +1,146 @@
+// !!! This is a file automatically generated by hipify!!!
+#include "hip/hip_runtime.h"
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <float.h>
+#include <ops_hip.cuh>
+
+#ifndef kernels
+#define kernels
+
+template <typename T>
+__global__ void
+    kEstimateQuantiles(T* __restrict__ const A, float* code, const float offset, const T max_val, const int n);
```

Comment:
nit: I've removed this from the CUDA version in https://github.com/bitsandbytes-foundation/bitsandbytes/pull/1669. We can remove from HIP implementation as well.